1-Draft/RFCNNNN-Native-Markdown-Rendering.md

```diff
@@ -0,0 +1,141 @@
+---
+RFC: 
+Author: Aditya Patwardhan
+Status: Draft
+SupercededBy: 
+Version: 0.1
+Area: Formatting and Output
+Comments Due: 03/26/2018
+Plan to implement: Yes
+---
+
+# Native Markdown Rendering
+
+Render markdown content on console to improve readability. 
+
+## Motivation
+
+Markdown is a common authoring format used by the community.
+There is no easy way in PowerShell to visualize a markdown document on console.
+
+## Specification
+
+This RFC proposes to use VT100 escape sequences to render markdown content.
+`ConvertFrom-Markdown` cmdlet as part of `Microsoft.PowerShell.MarkdownRender` PowerShell module would consume a string or a file path and output a VT100 encoded string.
+Optionally, the output can be formatted as HTML.
+For converting strings to VT100 coded strings, we will be writing an extension to [markdig](https://github.com/lunet-io/markdig).
```

Comment:
markdig already implement HTML renderer - nobody ask them about plans to implement VT100 renderer?

Reply:
markdig has a good extendable architecture and they prefer extensions. We may publish a nuget package of the extension we write. so it can be used by others.